gnd/build.gradle

```diff
@@ -62,10 +68,13 @@ android {
         release {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+            resValue("string", "GOOGLE_MAPS_API_KEY", googleMapsApiKey)
```

Comment:
Cool trick!

Reply:
This is a pretty huge change. Everyone who has cloned the project will have to create `secrets.properties` and add their API key.
I couldn't find a better method without a lot of disrupting code changes. Please suggest if you have a better solution.